ranges/subset.js

```diff
@@ -23,13 +31,16 @@ const compare = require('../functions/compare.js')
 // - If GT
 //   - If GT.semver is lower than any > or >= comp in C, return false
 //   - If GT is >=, and GT.semver does not satisfy every C, return false
+//   - If GT.semver has a prerelease, and not in prerelease mode
+//     - If no C has a prerelease and the GT.semver tuple, return false
 // - If LT
 //   - If LT.semver is greater than any < or <= comp in C, return false
 //   - If LT is <=, and LT.semver does not satisfy every C, return false
-// - If any C is a = range, and GT or LT are set, return false
+//   - If GT.semver has a prerelease, and not in prerelease mode
```

Comment:
I think this should be the following (could be wrong):

```suggestion
//   - If LT.semver has a prerelease, and not in prerelease mode
```